ci/prow/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM registry.fedoraproject.org/fedora:30
+FROM registry.svc.ci.openshift.org/coreos/cosa-buildroot:latest
 WORKDIR /src
 COPY . .
-RUN ./ci/build.sh
+USER root
+RUN env SKIP_INSTALLDEPS=1 ./ci/build-check.sh
```

Comment:
I'd rather not run any kind of tests during image build here. These might fail and the resulting image cannot be used for other tests (unless `make check` and `make rust-test` are very-very basic)

Reply:
They're pretty basic tests.

That said, there is already today that split, we can just run `ci/build.sh`.